com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/naming/QualifiedNameSegmentTreeLookup.java

```diff
@@ -284,6 +284,28 @@ public void accept(final Visitor visitor) {
       }
     }
 
+    public List<List<String>> getMappings(final Object value) {
+      List<List<String>> result = new ArrayList<>();
+      if (children != null) {
+        for (SegmentNode node : children) {
+          List<List<String>> childResult = node.getMappings(value);
+          for (List<String> mapping : childResult) {

```

Comment:
```suggestion
          for (List<String> mapping : node.getMappings(value)) {
```
and remove the local variable?